include-build/roborazzi-gradle-plugin/src/main/java/io/github/takahirom/roborazzi/WebAssets.kt

```diff
@@ -71,6 +64,17 @@ class WebAssets private constructor(private val webJarVersionLocator: WebJarVers
 
   companion object {
 
+   internal val assetPathsMap = mapOf(
```

Comment:
If we don't use this in tests or elsewhere, could we make this private?

```suggestion
   private val assetPathsMap = mapOf(
```

Reply:
oh, thought you'll need it else where for use. Thanks for the fix and making it `private`